src/persist-cli/src/maelstrom/txn_list_append_multi.rs

```diff
@@ -134,8 +134,8 @@ impl Transactor {
                         debug!("read ts {}", read_ts);
                         write_ts = self.oracle.write_ts();
                         debug!("write ts {}", write_ts);
-                        // TODO(txn): Read this incrementally between the old
-                        // and new read timestamps, instead.
+                        // TODO: Read this incrementally between the old and new
```

Comment:
This is more for OCC txn tuning, which has turned out to very much not be in scope for v0 of getting this out to prod